app/services/make_an_offer.rb

```diff
@@ -45,9 +45,10 @@ def save
           application_choice.offer = { 'conditions' => offer_conditions }
           application_choice.offered_at = Time.zone.now
           application_choice.save!
-          SetDeclineByDefault.new(application_form: application_choice.application_form).call
```

Comment:
Can we do away with the whole transaction here, as there's only a one database update in the statement now?

Reply:
:+1: Will update